includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -618,6 +618,40 @@ public function list_transactions( $page = 0, $page_size = 25, $sort = 'date', $
 		return $transactions;
 	}
 
+	/**
+	 * Initiates transactions export via API.
+	 *
+	 * @param int    $page       The requested page.
+	 * @param int    $page_size  The size of the requested page.
+	 * @param string $sort       The column to be used for sorting.
+	 * @param string $direction  The sorting direction.
+	 * @param array  $filters    The filters to be used in the query.
+	 * @param string $deposit_id The deposit to filter on.
+	 *
+	 * @return array Export summary
+	 *
+	 * @throws API_Exception - Exception thrown on request failure.
+	 */
+	public function get_transactions_export( $page = 0, $page_size = 25, $sort = 'date', $direction = 'desc', $filters = [], $deposit_id = null ) {
+		// Map Order # terms to the actual charge id to be used in the server.
+		if ( ! empty( $filters['search'] ) ) {
+			$filters['search'] = WC_Payments_Utils::map_search_orders_to_charge_ids( $filters['search'] );
+		}
+
+		$query = array_merge(
+			$filters,
+			[
+				'page'       => $page,
+				'pagesize'   => $page_size,
+				'sort'       => $sort,
+				'direction'  => $direction,
+				'deposit_id' => $deposit_id,
+			]
```

Comment:
Are these (`page`, `pagesize`, etc.) needed for the server endpoint export? It looks like they were removed from the query parameters in https://github.com/Automattic/woocommerce-payments/pull/2939/commits/9c560a63aecf2fb3de64d6f5dcceb97b8c0a8948.

Reply:
You're right. I removed these from the endpoint, and missed these when cleaning them up from this PR.

I've removed these in https://github.com/Automattic/woocommerce-payments/pull/2939/commits/3b05b8ed3923572926845374a2e79ede30750ab4